### ISSUE TITLE
Multi-line parameter comment sniffing

### ISSUE THREAD

**Issue by johnbillion** [WordPress-Coding-Standards/WordPress-Coding-Standards#407]
Given the following multi-line parameter comment, the "Parameter comment must end with a full stop" error is triggered on the first line.

```php
* @param array $args Hello hello hello hello hello hello hello hello
*                    hello.
```

What's WPCS's stance on multi-line parameter comments?

**Comment by GaryJones** [comment]
Testing with:

~~~php
 * @param bool $automatic Whether this is an automatic activation after an install. Defaults to false.
 *                        This determines the styling of the output messages.
~~~

I get no error  (correct). Removing the full stop after "messages", and the error correctly triggers.

Equally, testing with:

~~~php
/**
 * Summary.
 *
 * @param array $args Hello hello hello hello hello hello hello hello
 *                    hello.
 */
function x( $args ) {

}
~~~

I get no error - the full stop after the lonely hello is correctly recognised.

Also, FWIW, if documenting an array, the [hash notation](https://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/#1-1-parameters-that-are-arrays) should be used anyway, and we don't yet have the custom sniffs in place for them yet.

**Comment by johnbillion** [comment]
Gremlins.
